packages/cozy-client/src/hooks/useQuery.js

```diff
@@ -22,17 +22,8 @@ const generateFetchMoreQueryDefinition = queryResult => {
 /**
  * Fetches a queryDefinition and returns the queryState
  *
- * @param  {QueryDefinition} queryDefinition - Definition created with Q()
- *
- * @typedef {object} UseQueryOptions
- * @property  {object} as - Name for the query [required]
- * @property  {boolean} enabled - If set to false, the query won't be executed
- * @property  {object} fetchPolicy - Fetch policy
- * @property  {object} singleDocData - If true, the "data" returned will be
- * a single doc instead of an array for single doc queries. Defaults to false for backward
- * compatibility but will be set to true in the future.
- * @property {Function} onError - Callback if the query is errored
- *
+ * @param {QueryDefinition} queryDefinition - Definition created with Q()
+ * @param {UseQueryOptions} options - Options created with Q()
```

Comment:
`QueryOptions` is the right existing type, no?

Reply:
j'ai l'impression qu'il y a des similitudes mais aussi des différences. Est-ce que singleDocData est valable sur client.query, ainsi que enabled ? A l'inverse est-ce que autoUpdate est ok avec useQuery ?

Reply:
pour moi, il y a une sorte d'héritage oui => 

is it possible to set inheritance to QueryOptions?
https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/types.js#L77

because we give to client.query the options params https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/src/hooks/useQuery.js#L76